src/screens/Landing/index.js

```diff
@@ -168,8 +169,9 @@ export default function Landing() {
 
   const start = useCallback(() => {
     if (inspectionId && getInspection.state.loading !== true) {
-      getInspection.start().catch(() => {
-        // TODO: Add Monitoring code for error handling in MN-182
+      getInspection.start().catch((err) => {
+        // sentry code for error capturing
```

Comment:
you can remove this comment

Reply:
I think, we should keep this comment as it will help us to understand purpose of function, instead of reading the code and checking the implementation. What do you think?

Reply:
Done